themes/default/content/blog/pulumi-service-updates/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Pulumi Service Updates"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2022-07-14
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or social-media
+# previews. This field is required or the build will fail the linter test. 
+# Max length is 160 characters.
+meta_desc: Pulumi Service, the easiest way to use Pulumi’s open-source universal infrastructure as code, just got better with a new and improved user experience.
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - anita-trimbur
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - pulumi-service
+    - user-experience
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+Earlier this year, Pulumi's user experience (UX) team sat down with users to understand how we can better serve the community. It's important not only to the UX team, but across the company, that we listen to the real wants, needs, pains, and excitements of the people who use our product.
```

Comment:
suggest changing "better serve the community" to something more specific such as "how we can improve Pulumi"